Composer/packages/client/src/utils/getPageName.tsx

```diff
@@ -4,8 +4,8 @@
 import { PageNames } from '@bfc/shared';
 
 const PagesRegex = {
-  [PageNames.Design]: /\/dialogs/i,
-  [PageNames.Home]: /\/home/i,
+  [PageNames.Design]: /\/dialogs|\/bot\/d+.d+/i,
+  [PageNames.Home]: /\/|\/home|\/projects\/create|\/projects\/open/i,
```

Comment:
So this now includes project creation? Would it be better to separate the project creation pages into it's own page category?

Reply:
Per a conversation with @garypretty, since we are already capturing custom events for `/project/create` and `/projects/open` it would be redundant to capture them as a page view.